README.md

```diff
@@ -18,3 +18,11 @@ The initialization can be done by providing an `options` object such as:
     token: "sd1029bfwef9adas+0feqefi"
 }
 ```
+
+## Configuration
+
+| Name                         | Type  | Mandatory | Default                  | Description                                                      |
```

Comment:
Remove the mandatory field in the config, by definition all fields should be nonmandatory check the Appier configuration docs